azurerm/provider.go

```diff
@@ -156,6 +156,7 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_key_vault_certificate":               resourceArmKeyVaultCertificate(),
 			"azurerm_key_vault_key":                       resourceArmKeyVaultKey(),
 			"azurerm_key_vault_secret":                    resourceArmKeyVaultSecret(),
+			"azurerm_key_vault_policy":                    resourceArmKeyVaultPolicy(),
```

Comment:
I actually was expecting this one and told my team I didn't want to build support for this resource into anything until the review since I thought we would need to rename it.  I really should have named this azurerm_key_vault_access_policy from the start.  I apologize for not doing this.